config/locales/en.yml

```diff
@@ -13,6 +13,9 @@ en:
         trip:
           start_date_in_the_past: "Start date can not be in the past!"
           start_date_after_end_date: "Start date must be before end date!"
+        tripfamily:
+          trip_capability: "Check how manyy resources still to allocate!"
+          capability_exceeded: "The capability for this family is not enough!"
```

Comment:
Message is not clear. Maybe change: "Capability for this family has been exceeded."

Also, two suggestions to make it more user friendly:
1. Indicate on the page the maximum they can fill in
2. Only allow them to fill in up to the maximum